### DISCUSSION TITLE
Get the list of multiple selections of a MultiBrowser

### DISCUSSION THREAD

**Discussion by saflo76** [fltk-rs/fltk-rs#1350] [Q&A] [answered]
Hi, I'm using a MultiBrowser widget to support not only listing but also marking multiple items, but I don't see any specific method to get the list of marked (selected) ones, when trying to use the value() method I see that it's only intended to know the current selected line.
So I tried to call value() repeatedly to obtain list of all selected items but it only works the first time (then reports always zero).
Thank you!
![multiple_items](https://user-images.githubusercontent.com/107424432/216814817-dd0bea39-ea3d-44bf-b606-20dedeb71622.jpg)

**Comment by MoAlyousef** [comment, marked as answer]
Hi
It seems Browser operates on indices, I’ll try to provide a method which returns a `Vec<String>` of the selected items.

**Reply by MoAlyousef** [reply]
I've added a method BrowserExt::selected_items() which returns a Vec of indices:
```rust
        let s = b.selected_items();
        for i in s {
            println!("{:?}", b.text(i));
        }
```

**Comment by saflo76** [comment]
Thanks, in the end I have done it by removing item backwards to not subvert indices

`for i in scan_list.selected_items().iter().rev() { scan_list.remove(*i) }`
